packages/google_maps_flutter/google_maps_flutter_ios/example/ios14/integration_test/google_maps_test.dart

```diff
@@ -837,6 +846,26 @@ void main() {
     iwVisibleStatus = await controller.isMarkerInfoWindowShown(marker.markerId);
     expect(iwVisibleStatus, true);
 
+    // Update marker and check if the info window is still visible.
```

Comment:
I think it would be better to make this a separate test (`'updating a marker does not hide its info window'`) rather than adding it to this test. Yes, there will be some duplicate boilerplate, but the fewer distinct things a test is testing, the easier each test is to maintain, debug, and understand failure in.

Reply:
Oh, had this as separate test first, but desided to reduce test execution time as I saw this as part of info window toggle test.
Separated to own test as suggested.